src/screens/UnregisterToken.js

```diff
@@ -62,9 +69,13 @@ class UnregisterToken extends React.Component {
       return;
     }
 
-    const tokens = hathorLib.tokens.unregisterToken(this.props.selectedToken.uid);
-    this.props.dispatch(setTokens(tokens));
-    this.props.navigation.navigate('Dashboard');
+    const promise = hathorLib.tokens.unregisterToken(this.props.selectedToken.uid);
```

Comment:
As I commented on the lib changes, I don't like having a promise for now async operations.